### DISCUSSION TITLE
How to give Monika chocolates, coffee and the quetzal?

### DISCUSSION THREAD

**Discussion by ChocokrispisToto** [Monika-After-Story/MonikaModDev#9864] [general - spritepacks]
HELP: I don't know how to give Monika gifts that are in the same game files...

My case. It turns out that in the game I wanted to give Monika the coffee, the chocolates or the quetzal, and I went to the official page of the mod and downloaded the Spritepacks. but there were no gifts like the ones I mentioned above.
Investigating, it turns out that they are in the same game, but I do not know how to give it to them. I already saw tutorials and reviews all over the place, but it wasn't much.

If anyone knows that this comment and/or recommendation, please contact an administrator or moderator of the mod, to add the
coffee, the quetzal, candies, sweet corn, hot chocolate, fudge, Christmas cookies, candy canes, chocolates, thermos cup, roses and the ring **IN THE SPRITEPACKS**, that would be very helpful. since many of us do not know how to "code" some files.

PD: I'm from windows 7

**Comment by Booplicate** [comment]
> and the ring IN THE SPRITEPACKS

Those are not spritepacks, so they won't be included with spritepacks. To create a gift, you just need to make a new file, [here's the guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide), you don't code anything.

**Reply by ChocokrispisToto** [reply]
I read all the steps to give him the gifts. I did all the steps from head to toe and it has not worked for me. 
Thanks anyway for the help...
:(

**Reply by Booplicate** [reply]
Then you did something wrong. Show a screenshot of one of your gifts.

**Reply by ChocokrispisToto** [reply]
ok
-Before doing the steps to give Monika the gifts, you first have to make the text document end in ".txt". Ready
*
Step 1. Create a new text document in "characters".
Step 2. Rename the text with the Gift ID (in this case, "coffee").
Step 3. After the point, instead of saying ".txt", it will be called ".gift". and I give him to accept.
Step 4. Then open the app and wait for the gift. and that's it

LOL

![Captura](https://user-images.githubusercontent.com/122856596/213285994-47833206-1a7e-4146-a2bf-f9fa6fae02fe.PNG)
![image](https://user-images.githubusercontent.com/122856596/213286072-efb3c5d3-affd-412c-8f90-c9a77a20f754.png)

**Reply by Booplicate** [reply]
Are you using a translation of the mod?

**Comment by ChocokrispisToto** [comment]
[image: image.png][image: image.png]


-Before doing the steps to give Monika the gifts, you first have to make
the text document end in ".txt". Ready
*
Step 1. Create a new text document in "characters".
Step 2. Rename the text with the Gift ID (in this case, "coffee").
Step 3. After the point, instead of saying ".txt", it will be called
".gift". and I give him to accept.
Step 4. Then open the app and wait for the gift. and that's it
